nexum_ai/tests/README.md

```
@@ -1,172 +1,172 @@
+```bash
+pytest -v
+```
+
+### Run tests excluding slow tests:
+```bash
+pytest -m "not slow"
+```
+
+## Test Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)